views/components/Certification.js

```diff
@@ -1,72 +1,396 @@
+        </Header>
+        <p>
+          The TOEFL iBT® test measures your ability to use and understand
+          English at the university level. And it evaluates how well you combine
+          your reading, listening, speaking and writing skills to perform
+          academic tasks.
+        </p>
+      </DevelopInfo>
+    </Fragment>
+  )
+}
+
+const Deploy = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <Fragment>
+      <DeployItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[2])}>
+        <ItemIcon
+          id={INIT_CERT_ACTIVE_ITEM[2]}
+          itemSelected={itemSelected}
+          name="p"
+        />
+      </DeployItem>
+
+      <DeployInfo itemSelected={itemSelected}>
+        <Header>
+          <span>3</span> &mdash; Second place
+        </Header>
+        <p>
+          Got second prize for the competitive algorithm contest of the
+          university
+        </p>
+      </DeployInfo>
+    </Fragment>
+  )
+}
+
+const Deliver = ({ itemSelected, setSelectedItem }) => {
+  return (
+    <Fragment>
+      <DeliverItem onClick={() => setSelectedItem(INIT_CERT_ACTIVE_ITEM[3])}>
+        <ItemIcon
+          id={INIT_CERT_ACTIVE_ITEM[3]}
+          itemSelected={itemSelected}
+          name="p"
+        />
+      </DeliverItem>
+
+      <DeliverInfo itemSelected={itemSelected}>
+        <Header>
+          <span>4</span> &mdash; First place
+        </Header>
+        <p>
+          Highschool, got First prize for the competitive algorithm contest of
+          the city.
+        </p>
+      </DeliverInfo>
+    </Fragment>
+  )
+}
+
+const ItemIcon = ({ id, itemSelected, name }) => {
+  return itemSelected == id ? (
+    <ActiveIcon name={name} />
+  ) : (
+    <InactiveIcon name={name} />
+  )
+}
+
+Research.propTypes = {
+  itemSelected: PropTypes.any,
+  setSelectedItem: PropTypes.func
+}
+
+Develop.propTypes = {
+  itemSelected: PropTypes.any,
+  setSelectedItem: PropTypes.func
+}
+
+Deploy.propTypes = {
+  itemSelected: PropTypes.any,
+  setSelectedItem: PropTypes.func
+}
+
+Deliver.propTypes = {
+  itemSelected: PropTypes.any,
+  setSelectedItem: PropTypes.func
+}
+
+ItemIcon.propTypes = {
+  id: PropTypes.number,
+  itemSelected: PropTypes.any,
+  name: PropTypes.string
+}
+
+const Container = styled.section``
+
+const List = styled.ul`
+  position: relative;
+  list-style-type: none;
+  height: 360px;
+`
+
+const Item = styled.li`
+  position: absolute;
+  top: 0;
+  display: block;
+  width: 25%;
+  height: 100%;
+  background-color: rgba(255, 255, 255, 0);
+  background-repeat: repeat;
+  margin-left: -12.5%;
+`
+
+const ResearchItem = styled(Item)`
+  left: 0%;
+  margin-left: 0%;
+  z-index: 40;
+`
+
+const DevelopItem = styled(Item)`
+  left: 37.5%;
+  z-index: 30;
+`
+
+const DeployItem = styled(Item)`
+  left: 62.5%;
+  z-index: 20;
+`
+
+const DeliverItem = styled(Item)`
+  left: 87.5%;
+  z-index: 10;
+`
+
+const CertIcon = styled(Icon)`
```

Comment:
Refractor these Icon to other folders

Reply:
Moved these Icon to a new file CertIcon.js in /views/components/common

![3](https://user-images.githubusercontent.com/7389488/70732398-f1303b00-1d3a-11ea-9bbd-cb1ae2318e3c.PNG)

Reply:
no CertIcon should be in sub folder for Certification may be views/components/certification/ 👯‍♂

Reply:
Updated